### PR TITLE
Add tests for mean.jl

### PR DESCRIPTION
**Pull request by sayantikaSSG**
Add test cases to the mean.jl file according to #103  as stated in the following comment:

> [Codedev page for mean.jl](https://app.codecov.io/gh/xKDR/Survey.jl/blob/main/src/mean.jl) illustrates which of the mean() functions tests have to be added for.

> line 64, `mean(x::Symbol,design::StratifiedSample)`

> line 127, `mean(x::Symbol, by::Symbol, design::SimpleRandomSample)`

> line 142, `mean(x::Symbol, by::Symbol, design::StratifiedSample)`

### CODE REVIEW

File: test/mean.jl [hunk 1]
```diff
@@ -39,15 +39,40 @@
...
+    apistrat_original = load_data("apistrat")
+    apistrat = copy(apistrat_original)
+    strat = StratifiedSample(apistrat, :stype; popsize = :fpc)
+    mean_strat = mean(:api00, strat)
+    @test mean_strat.mean[1] ≈ 662.287 atol = 1e-2
+    @test mean_strat.SE[1] ≈ 9.40894 atol = 1e-2
 end
 
 @testset "mean_svyby_SimpleRandomSample" begin
     ## Add tests
```
Comment (ayushpatnaikgit) [inline test/mean.jl:51]: No need to say add tests. 
```suggestion
```

File: test/mean.jl [hunk 2]
```diff
@@ -39,15 +39,40 @@
...
+    @test mean_symb_srs.mean[1] ≈ 605.36 atol = 1e-2
+    @test mean_symb_srs.mean[2] ≈ 666.141 atol = 1e-2
+    @test mean_symb_srs.mean[3] ≈ 654.273 atol = 1e-2
+    @test mean_symb_srs.SE[1] ≈ 21.9266 atol = 1e-2
+    @test mean_symb_srs.SE[2] ≈ 11.1935 atol = 1e-2
+    @test mean_symb_srs.SE[3] ≈ 21.8261 atol = 1e-2
 end
 
 @testset "mean_svyby_Stratified" begin
     ## Add tests
```
Comment (ayushpatnaikgit) [inline test/mean.jl:65]: No need to say add tests. 
```suggestion
```

**Review by smishr — APPROVED** [review]
looks good to add

**Comment by smishr** [comment]
i am merging the tests and closing #103

**Pull request merged**